spec/views/users/user_index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
```

Comment:
- [ ] **Required:** Some required tests are missing:

- It should have a test to check that the user can redirect to the posts they made.

Reply:
well resolved! ✔️